web_ui/src/pages/annotator/hooks/use-grabcut.hook.ts

```diff
@@ -36,7 +35,7 @@ const convertGetiDataToGrabcutData = (data: GrabcutData): ToolGrabcutData => {
 
 export const useGrabcut = ({ showNotificationError, onSuccess }: useGrabcutProps): useGrabcutResult => {
     const { worker, isLoading } = useLoadAIWebworker(AlgorithmType.GRABCUT);
-    const grabcutRef = useRef<Remote<Grabcut> | null>(null);
+    const grabcutRef = useRef<Grabcut | null>(null);
```

Comment:
Why was this removed? Shouldn't the Grabcut instance live in a web worker and thus be remote?